snuba/util.py

```diff
@@ -275,3 +276,16 @@ def wrapper(*args: Any, **kwargs: Any) -> Any:
         return cast(F, wrapper)
 
     return decorator
+
+
+@dataclass
+class MergeInfo:
+    partition_id: str
```

Comment:
I think you can remove `partition_id` from this dataclass since you don't use this anywhere.

Reply:
removed